server/src/integration-test/java/com/scalar/db/server/DistributedStorageAdminServiceRepairTableIntegrationTest.java

```diff
@@ -60,6 +65,14 @@ public void repairTable_ForTruncatedMetadataTable_ShouldRepairProperly() throws
     super.repairTable_ForTruncatedMetadataTable_ShouldRepairProperly();
   }
 
+  /** This test is disabled if {@link #isExternalServerUsed()} return true */
+  @Override
+  @Test
+  @DisabledIf("isExternalServerUsed")
+  public void repairTable_ForCorruptedMetadataTable_ShouldRepairProperly() throws Exception {
```

Comment:
@Torch3333 Ditto. I think we also need to disable this method if isExternalServerUsed is true, right?

Reply:
Yes, thank you for fixing this.